### PR TITLE
 Enhanced the UI of LITERARY REALMS SECTION

### PR DESCRIPTION
**Pull request by DevROHIT11**
# Related Issue
None

Fixes:  #4117

# Description

I was assigned to task to enhance the literary realms section and i succesfully completed the task  by making changes like adding hover effect changing the box size making text visible ,etc.

# Type of PR

- [ ] Bug fix
- [ x] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

# Screenshots / videos (if applicable)

https://github.com/user-attachments/assets/c075d6cc-a473-4746-9eb5-8def714ed0a2

# Checklist:

- [x ] I have made this change from my own.
- [ ] I have taken help from some online resources.
- [x ] My code follows the style guidelines of this project.
- [ x] I have performed a self-review of my own code.
- [ x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x ] My changes generate no new warnings.
- [ x] I have tested the changes thoroughly before submitting this pull request.
- [ x] I have provided relevant issue numbers and screenshots after making the changes.

### CODE REVIEW

**Comment by DevROHIT11** [comment]
@abhi03ruchi  please accept my pull request before the merge conflicts occur 
thank you !!

**Comment by abhi03ruchi** [comment]
chk deploy preview . there is some bug . 


https://github.com/user-attachments/assets/cf1d5b68-0c57-4623-a2df-5e17a17f3b52

fix it.

**Comment by abhi03ruchi** [comment]
work on updated branch .

File: assets/css/style.css [hunk 1]
```diff
@@ -1277,14 +1282,19 @@ play-btn ion-icon {
...
-  background-color: pink;
+  background-color: rgba(255, 202, 211, 0.67);
   /* Pink line color */
   transition: width 0.3s ease;
+  transition: all cubic-bezier(0.075, 0.82, 0.165, 1);
   /* Transition width property */
 }
 
 .chapter-card:hover {
+  background-color:rgba(255, 192, 203, 0.603);
```
Comment (yashksaini-coder) [inline assets/css/style.css:1293]: The card hover is not rendering properly I guess

**Comment by DevROHIT11** [comment]
@abhi03ruchi can you please  tell me what actually the bug is !!

File: assets/css/style.css (hunk 1 above)
Comment (yashksaini-coder) [reply assets/css/style.css:1293]: @abhi03ruchi is that the bug you were mentioning in the above video

**Comment by abhi03ruchi** [comment]
> @abhi03ruchi can you please tell me what actually the bug is !!

chk ur css code . style.css .

the homepage is like this : -- 

Screenshot 2024-11-04 at 5 20 53 PM

**Comment by DevROHIT11** [comment]
> > @abhi03ruchi can you please tell me what actually the bug is !!
> 
> chk ur css code .
> 
> the homepage is like this : --  but in ur deployment it isnt like that . also chk the genre section . 
> 
> Screenshot 2024-11-04 at 5 20 53 PM
> 

Okk i will check that

**Pull request closed without merging**